cairo/src/enums.rs

```diff
@@ -2109,26 +2631,44 @@ impl fmt::Display for ScriptMode {
     }
 }
 
+/// Specifies the type of a given [`Device`], also known as "backends" within Cairo.
 #[derive(Clone, PartialEq, Eq, PartialOrd, Debug, Copy)]
 #[non_exhaustive]
 #[doc(alias = "cairo_device_type_t")]
 pub enum DeviceType {
+    /// Type Direct Render Manager.
     #[doc(alias = "CAIRO_DEVICE_TYPE_DRM")]
-    Ascii,
+    Drm,
+
+    /// Type OpenGL.
     #[doc(alias = "CAIRO_DEVICE_TYPE_GL")]
-    Binary,
```

Comment:
The names of the variants here don't match the names in the C documentation / doc alias! This could confuse users of the library.

Reply:
Would you mind fixing this one?